testsuite/MDAnalysisTests/data/coordinates/test.gro

```diff
@@ -0,0 +1,8 @@
+Written by MDAnalysis
+    5
+    1MET     CA    1   0.000   1.600   3.200  0.0000  0.1600  0.3200
+    2ARG     CA    2   4.800   6.400   8.000  0.4800  0.6400  0.8000
+    3ILE     CA    3   9.600  11.200  12.800  0.9600  1.1200  1.2800
+    4LYS     CA    4  14.400  16.000  17.600  1.4400  1.6000  1.7600
+    5LEU     CA    5  19.200  20.800  22.400  1.9200  2.0800  2.2400
+   8.51000   8.59223   8.35003   0.00000   0.00000   0.69131   0.00000   1.45589   2.09054
```

Comment:
This looks like it's only the last frame. But according to the docs GRO can only write one frame per file. Multiple frames are stored in multiple files. So the trajectory in gro files can only be read with the chain-reader (that works and it tested separately no need to test the chain reader in this PR)